src/ray/core_worker/transport/sequential_actor_submit_queue.cc

```diff
@@ -101,8 +101,8 @@ uint64_t SequentialActorSubmitQueue::GetSequenceNumber(
   return task_spec.ActorCounter() - caller_starts_at;
 }
 
-void SequentialActorSubmitQueue::MarkTaskCompleted(uint64_t sequence_no,
-                                                   const TaskSpecification &task_spec) {
+void SequentialActorSubmitQueue::MarkSeqnoCompleted(uint64_t sequence_no,
```

Comment:
Can you update this protocol change to sequential submission queue docstring? Basically, we can say if task is retried for exceptions, we mark the seqno is finished (so that we can refresh the seqno)

Reply:
done.